Shoko.Server/Mappings/VideoLocal_UserMap.cs

```diff
@@ -13,7 +13,9 @@ public VideoLocal_UserMap()
             Map(x => x.JMMUserID).Not.Nullable();
             Map(x => x.VideoLocalID).Not.Nullable();
             Map(x => x.WatchedDate);
+            Map(x => x.WatchedCount).Not.Nullable();
             Map(x => x.ResumePosition).Not.Nullable();
+            Map(x => x.DateTimeUpdated).Not.Nullable();
```

Comment:
Suggestion: rename this to `LastUpdated` as that is typically the naming format for these columns

Reply:
I just stole the name from what was already used in the other models/tables. It can still be renamed.